code/modules/reagents/chems/chems_nutriment.dm

```diff
@@ -347,3 +347,25 @@
 	exoplanet_rarity_plant = MAT_RARITY_NOWHERE
 	exoplanet_rarity_gas = MAT_RARITY_NOWHERE
 	uid = "chem_nutriment_mayonnaise"
+
+/decl/material/liquid/nutriment/cheese
+	name = "cheese"
+	lore_text = "Aged, fermented, curdled milk."
+	uid = "chem_nutriment_cheese"
+	//allergen_flags = INGREDIENT_FLAG_DAIRY
```

Comment:
should probably generalize this to an `ingredient_flags` var and fold soup's `soup_flags` into it

Reply:
also, no taste description? assume it's overridden when it's created

Reply:
yeah i just meant in the future, wasn't meant as a request, i originally intended to approve when writing this comment